src/modals/_SearchModal.jsx

```diff
@@ -1,28 +1,30 @@
 import React, { useState, useEffect, useRef } from 'react';
 import PropTypes from 'prop-types';
-import SearchField from 'terra-search-field';
-import ContentContainer from 'terra-content-container';
-import ActionHeader from 'terra-action-header';
+import { useHistory } from 'react-router-dom';
+import ApplicationModal from '@cerner/terra-application/lib/application-modal/ApplicationModal';
+import classNamesBind from 'classnames/bind';
 import InfiniteList, { Item } from 'terra-infinite-list';
-import { DisclosureManagerContext } from 'terra-application/lib/disclosure-manager';
+import SearchField from 'terra-search-field';
+import { ThemeContext } from '@cerner/terra-application/lib/theme';
 import Fuse from 'fuse.js';
 import StatusView from 'terra-status-view';
-import classNames from 'classnames/bind';
-import { ThemeContext } from 'terra-application/lib/theme';
-import styles from './search.module.scss';
 
-const cx = classNames.bind(styles);
+import { pageConfigPropType } from '../site/siteConfigPropTypes';
+
+import styles from './SearchModal.module.scss';
+
+const cx = classNamesBind.bind(styles);
```

Comment:
I might be missing it, but is `search-field` the only css class being applied from https://github.com/cerner/terra-dev-site/blob/2adbedcfec6b91f2ccbb2135d5fe3d4fdf56b941/src/modals/SearchModal.module.scss#L5?